src/engraving/rendering/dev/paint.cpp

```diff
@@ -57,6 +57,7 @@ void Paint::paintScore(draw::Painter* painter, Score* score, const IScoreRendere
 
     // Setup Painter
     painter->setAntialiasing(true);
+    painter->setClipping(false);
```

Comment:
It looks like this change makes the condition `if (!painter->hasClipping()) {` in line +135 always true, which is remarkable. So we need to think carefully about that. It might undo the benefits of https://github.com/musescore/MuseScore/commit/f642ca7093333bf9a129fae393ce5c1ac311dc7b.

Reply:
It won't be if the painter doesn't support clipping...
At the moment that condition is always *false*, so it never applies the correct clipping.

Reply:
Actually it's worse than that, it appears this change has broken repainting while adding notes etc.  Looking into it.

Reply:
Ok have re-done.  It seems to be working OK for the cases I can test now, and the logic is simpler (no ifs!).